apps/explorer/src/app/App.tsx

```diff
@@ -91,6 +65,7 @@ function App() {
                         />
 
                         <ReactQueryDevtools />
+                        <Analytics />
```

Comment:
I'm just pulling this in since it's offered by the platform and would be useful to see what kind of data this provides us.